packages/java/src/browser/java-client-contribution.ts

```diff
@@ -80,7 +80,7 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
     protected onReady(languageClient: ILanguageClient): void {
         languageClient.onNotification(ActionableNotification.type, this.showActionableMessage.bind(this));
         languageClient.onNotification(StatusNotification.type, this.showStatusMessage.bind(this));
-        languageClient.onRequest(ExecuteClientCommand.type, params => this.commandService.executeCommand(params.command, ...params.arguments));
+        languageClient.onRequest(ExecuteClientCommand.type, params => this.commandService.executeCommand(params.command, ...params.arguments || []));
```

Comment:
Why do we need to change it? Does it mean a breaking change for all JSON-RPC using varargs? Could we avoid it?

Reply:
Switching compilation target to ES6 made TypeScript detect that `params.arguments` can be undefined meaning that it could not be unwrapped.

It is breaking if we try to unwrap undefined values in other places.